spk/ffsync/Makefile

```diff
@@ -1,47 +1,55 @@
 SPK_NAME = ffsync
-SPK_VERS = 1.5.2
-SPK_REV = 3
+SPK_VERS = 0.14.1
+SPK_REV = 4
 SPK_ICON = src/ffsync.png
-BETA = 1
 
-BUILD_DEPENDS = cross/python2 cross/setuptools_py2 cross/pip_py2 cross/wheel
-BUILD_DEPENDS += cross/gevent cross/greenlet cross/cffi
+BUILD_DEPENDS = cross/curl cross/mysql-connector-c
+DEPENDS     = cross/syncstorage-rs
 
-DEPENDS     = cross/busybox
-WHEELS      = src/requirements.txt
+PYTHON_PACKAGE = python311
+WHEELS      = src/requirements-crossenv.txt src/requirements-pure.txt
+
+REQUIRED_MIN_DSM = 6.0
+SPK_DEPENDS = MariaDB10:$(PYTHON_PACKAGE)
 
 MAINTAINER   = SynoCommunity
 DESCRIPTION  = Firefox Sync Server 1.5, used for Firefox 29 and later. You can use Firefox Sync Server to synchronize your bookmarks, passwords, settings, history, add-ons and tabs with Firefox on other computers. The service runs on port 8132.
 DISPLAY_NAME = Firefox Sync Server 1.5
-CHANGELOG = "1. Fix Python requirement<br>2. Fix startup issue when pidfile is not yet written"
+CHANGELOG = "1. Upgrade to Mozilla Sync Storage built with Rust."
 
-HOMEPAGE     = https://docs.services.mozilla.com/howtos/run-sync-1.5.html
-LICENSE      = MPL/GPL/LGPL
+HOMEPAGE     = https://mozilla-services.readthedocs.io/en/latest/howtos/run-sync-1.5.html
+LICENSE      = MPL-2.0
 HELPURL      = https://github.com/SynoCommunity/spksrc/wiki/Firefox-Sync-Server-1.5
```

Comment:
Description still refer to the older firefox sync server 1.5
Also homepage and helpurl seems to still refer to the older version 1.5

Reply:
My understanding is that this is still Sync Server 1.5 since the protocol it accepts from the client is 1.5. The build however got reset since it's now built in rust. The new repo still refers to the old docs and couldn't find a better place to reference.